docs/pages/database-access/guides/aws-dynamodb.mdx

```diff
@@ -0,0 +1,265 @@
+---
+title: Database Access with AWS DynamoDB
+description: How to access AWS DynamoDB with Teleport Database Access
+---
+
+Access to AWS DynamoDB can be provided by [Teleport Database
+Access](../introduction.mdx) for the AWS Console and API. This allows for
```

Comment:
```suggestion
Access](../introduction.mdx). This allows for
```
Same comment as below: This does not provide access to AWS Console/API in general, only DynamoDB specifically.